### PR TITLE
Add filter_branch for commando.io service

### PR DESCRIPTION
**Pull request by aw**
Hi :octocat:

This PR adds a form field to allow branch filtering in the service configuration. Mostly copied from `twitter.rb` ;)

Thanks!

### CODE REVIEW

**Comment by aw** [comment]
Sorry! I added a small documentation update :smile:

**Comment by nodesocket** [comment]
Any eta on when this will be merged in?

**Comment by nodesocket** [comment]
Following up again, a monthly later. Anyway to get this merged? Thanks.

**Comment by jechristianco** [comment]
Thank you very much I'm not sure are open all the help I can get

**Comment by kdaigle** [comment]
cc @ptoomey3 @gregose for review

**Comment by nodesocket** [comment]
@kdaigle any update? On merging this?

**Comment by kdaigle** [comment]
cc @ptoomey3 @gregose for review to get this :ship:

File: lib/services/commandoio.rb [hunk 1]
```diff
@@ -58,6 +64,17 @@ def receive_event
...
 
+  # Checks if branch filtering is enabled and if the payload's ref matches
+  #
+  # Returns true or false
+  def filter_branch?
+    commit_branch = (payload['ref'] || '').split('/').last || ''
+    filter_branch = data['filter_branch'].to_s
+
+    # If filtering by branch then don't make a post
+    ((filter_branch.length > 0) && (commit_branch.index(filter_branch) == nil)) ? false : true
```
Comment (ptoomey3) [inline lib/services/commandoio.rb]: I somewhat prefer the muliple/wildcard matching as is done in https://github.com/github/github-services/pull/1058/files#diff-5dc7176f8aa22691f0ae7be3dc7c2785R32. But, I don't have a strong opinion. :+1: so long as @kdaigle doesn't have a preference either.

**Comment by aw** [comment]
Oh, yeah I can fix that.

**Comment by kdaigle** [comment]
@aw can you get your fix in? Then I can merge this and get it deployed. :cake:

**Comment by pengwynn** [comment]
Friendly bump @aw. :wave:

**Comment by kdaigle** [comment]
Hey @aw. I'm going to close this since we haven't heard from you in a while. We definitely recommend moving towards a webhook, if at all possible, too. :sparkles:

**Pull request closed without merging**